<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Panaderia GLORIA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_agregar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>92</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Producto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_comprar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprar Productos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mostrar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar Productos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salir">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border: none;</string>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ganacias Hasta el Momento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #3fe70e;</string>
    </property>
    <property name="text">
     <string>$ 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
